File: Python-Programing-Basics/Nested Loops/Nested Loops - Lab/06_building/building.py
floors = int(input())
rooms = int(input())

for floor in range(floors, 0, -1):
    if floor == floors:
        letter = 'L'
    elif floor % 2 == 0:
        letter = "O"
    else:
        letter = "A"
    for room in range(0, rooms):
        print(f"{letter}{floor}{room}", end=" ")
    print()

File: JavaScript/Programming Basics - JavaScript/Nested Loops/building.js
function building(input){
    let floors = Number(input[0]);
    let rooms = Number(input[1]);
    for (let f = floors; f >= 1; f--){
        let allRooms = '';
        for (let r = 0; r < rooms; r++){
            if (f===floors){
                allRooms += `L${f}${r} `;
            } else if (f % 2 === 0){
                allRooms += `O${f}${r} `;
            } else if(f % 2 !== 0){
                allRooms += `A${f}${r} `;
            }
        }
        console.log(allRooms);
    }
}

building(["6", "4"]);